README.md

```diff
@@ -244,7 +244,7 @@ const updateRequest: UpdateRequest = {
     indices: [15, 30, 11],
     values: [0.1, 0.2, 0.3],
   }, // optional sparse values
-  setMetadata: metadata, // the new metadata
+  metadata: metadata, // the new metadata
```

Comment:
I'm planning to rewrite this entire README ahead of the release. i don't want people to find instructions for unpublished code in the README. But I appreciate the instinct to update docs as you go.